pkgs/development/cuda-modules/generic-builders/multiplex.nix

```diff
@@ -127,6 +127,8 @@ let
             inherit pname;
             redistName = pname;
             inherit (shims) redistribRelease featureRelease;
+            # get `autoAddDriverRunpath` from pkgs instead of cudaPackages' alias to avoid warning
+            inherit (final.callPackage ({ pkgs }: pkgs) { }) autoAddDriverRunpath;
```

Comment:
Why such an indirect way to refer to `pkgs.autoAddDriverRunpath`?

Reply:
I just copied it from https://github.com/NixOS/nixpkgs/commit/ae5074ef871e2c249b2421272d7959d59c8a5e6d

@wrvsrx Can you explain why you did it this way in that commit?

Reply:
Good enough for me for 24.05, given that this whole thing is no longer necessary for 24.11.

@SomeoneSerge Good to merge?

Reply:
sorry for blacking out, yes it's alright